undertale/models/item/finetune_embedding.py

```diff
@@ -51,147 +83,116 @@
     parser.add_argument(
         "--start-epoch", type=int, default=0, help="starting epoch number"
     )
-    parser.add_argument("-b", "--batch-size", type=int, default=8, help="batch size")
 
     arguments = parser.parse_args()
 
     undertale_logging.setup_logging()
 
-    os.makedirs(arguments.output, exist_ok=True)
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    if torch.cuda.is_available():
+        torch.cuda.set_device(0)
```

Comment:
Does this limit us to training on a single device? I think Lightning is supposed to just handle this for us